stackslib/src/config/mod.rs

```diff
@@ -3953,7 +3925,7 @@ pub struct FeeEstimationConfigFile {
     ///   by adding unpredictability to fee estimates while still maintaining accuracy.
     /// ---
     /// @default: `"ScalarFeeRate"`
-    /// @notes:
+    /// @notes: |
```

Comment:
I see you've added the pipe literal in a lot of notes, with the notes being a list of items. but, per this: https://github.com/Jiloc/stacks-core/tree/feat/config-docs-generator/contrib/tools/config-docs-generator#complete-example

i think if the `@notes` is a list it's not needed, correct? i read the accompanying syntax doc as this would only be needed if hte literal text formatting is needed for the output.